pulser-core/requirements.txt

```diff
@@ -1,5 +1,5 @@
 jsonschema
-matplotlib
+matplotlib >= 3.6.2
```

Comment:
This breaks installation for Python 3.7. Can't we avoid all these version requirements by making the plotting happen on custom axes rahter than custom figures?

Reply:
Thanks for the comment! It also works, it is simpler, and even more flexible because it allows to draw other plots on the register.

Reply:
The only downside I can see is that the `draw` function won't be able to switch to more complex plotting now, combining multiple plots. Let's hope it won't be needed.